src/utils/dappMailer.js

```diff
@@ -0,0 +1,272 @@
+const rp = require('request-promise');
+const _ = require('lodash');
+import { utils } from 'web3';
+
+const _sendEmail = (app, data) => {
+  // add the dapp url that this feathers serves for
+  _.extend(data, { dappUrl: app.get('dappUrl')});
+
+  console.log('send notification', data);
```

Comment:
can we remove this?

Reply:
why? I'd like to log which emails are being requested to sent